roles/docker-garbage-collection/defaults/main.yml

```diff
@@ -2,11 +2,15 @@
 # file path and name variables
 garbage_home: "/opt/docker/docker_garbage_collection"
 backup_images_dir: "backup_images"
-garbage_script: "garbage-collect-docker.sh"
+garbage_script: "garbage-collect.sh"
 backup_removal_script: "backup-removal.sh"
-backup_permission_script: "backup-removal-permission.sh"
 log_file: "docker_garbage_collection.log"
 
+# user and permission parameters
+garbage_user: "osu_apis"
+garbage_group: "apis"
```

Comment:
Could we just use some generic values like `user` and `password` for `garbage_user` and `garbage_group`? I mainly just wanna avoid any confusion. In addition, can you also add a playbook to this [repo](https://github.sig.oregonstate.edu/ecs-data/ansible-playbooks) to show how to use these roles?